components/Loading.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import PlateWheel from "/public/icons/svg/plate-wheel.svg";
+
+export default function LoadingComponent() {
+  return <RotatingSVG />;
+}
```

Comment:
```suggestion
```
when the styled component doesn't have any logic attached to it, you can export it directly and don't need to wrap it in a function :)